src/main/java/twitterapp/src/models/TwitterPost.java

```diff
@@ -0,0 +1,58 @@
+package twitterapp.src.models;
```

Comment:
add coverage

Reply:
you have your own java object, there are specific methods that you need to override for your own classes, please try looking for them and implementing them